pkg/volume/iscsi/iscsi_util_test.go

```diff
@@ -69,3 +71,39 @@ func TestExtractPortalAndIqn(t *testing.T) {
 		t.Errorf("extractPortalAndIqn: got %v %s %s", err, portal, iqn)
 	}
 }
+
+func fakeOsStat(devicePath string) (fi os.FileInfo, err error) {
+	var cmd os.FileInfo
+	return cmd, nil
+}
+
+func fakeFilepathGlob(devicePath string) (globs []string, err error) {
+	return []string{devicePath}, nil
+}
+
+func TestWaitForPathToExist(t *testing.T) {
+	devicePath := []string{"/dev/disk/by-path/ip-127.0.0.1:3260-iqn.2014-12.com.example:test.tgt00-lun-0",
+		"/dev/disk/by-path/pci-0000:00:00.0-ip-127.0.0.1:3260-iqn.2014-12.com.example:test.tgt00-lun-0"}
+	osStat = fakeOsStat
+	defer func() { osStat = os.Stat }()
```

Comment:
@swagiaal I'm a little lost here as to how to proceed here, but maybe I just need to double check.  newBuilderInternal is not accepting varying parameter types. `os.Stat()` and `filepath.Glob()` have different signatures, `(type func(string) (os.FileInfo, error))` vs `(type func(string) ([]string, error))` so I'm not sure how to pass them to the same function (unless a nil check is acceptable, and it takes both as separate parameters)